Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -17,30 +17,38 @@
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
-        <item>Registrierung von PCR-Schnelltests</item>
-        <item>Zuordnung von Zertifikaten </item>
-        <item>Zertifikate aktualisieren</item>
+        <item>Anzeige erhöhten Risikos verkürzt</item>
+        <item>Risikoanzeige nach positivem Testergebnis</item>
+        <item>Nachricht zu neuer Risikobegegnung</item>
```

Comment:
The headline is wrong and does not fit to the description. Please verify.

Reply:
In iOS it's "Nachricht über Status-Änderung": https://github.com/corona-warn-app/cwa-app-ios/blob/848bc38b7bfa9906e2c55e9421d2b7ec49f9b599/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L1667

Reply:
@KathrinLuecke , @CV113  FYI

Reply:
There should be both headlines
item 3 : "Nachricht zu neuer Risikobegegnung" and item 4: "Nachricht über Statusänderung"
I see however, that item 4 is missing in Figma. I will add it asap.